drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/referral_posting.rb

```diff
@@ -29,6 +32,18 @@ class ReferralPosting < ::HmisExternalApis::HmisExternalApisBase
         assigned_to_other_program_status: 60,
         # closed: 65,
       },
+      referral_result: {
+        client_accepted_referral_result: 1,
+        client_rejected_referral_result: 2,
+        provider_rejected_referral_result: 3,
+      },
```

Comment:
I don't love having these listed out, since it's a HUD enum that could change in future specifications. Can this be replaced with `::HudLists.referral_result_map.invert`? The `hud_lists.rb` file generated based of `hud_lists.json` that's our source-of-truth for HUD mappings.

Reply:
Thanks, I fiddled with that map to make the enum labels snake-case like rails expects